ord.md

```diff
@@ -2,22 +2,22 @@
 
 The cap-js ORD plugin has a default behavior where all the values of the ORD document are generated automatically by the plugin without the user giving them explicitly.
 
-If this default behavior is to be overwritten , then there are two ways in which it could be done.
+If this default behavior is to be overwritten, then there are two ways in which it could be done.
 
-For all the application level felids it is done using `present configuration` ( in `./cdsrc.json` under `ord` section).
+For all the application global information it can be done using `present configuration` ( in `./cdsrc.json` under `ord` section).
```

Comment:
```suggestion
For all the application global information, it can be done using `present configuration` ( in `./cdsrc.json` under `ord` section).
```